### DISCUSSION TITLE
Large mod collection merged and conflicts resolved... and then disappeared

### DISCUSSION THREAD

**Discussion by DiettingerX** [bcssov/IronyModManager#246] [Q&A]
Is there a situation which causes the conflict resolutions to disappear from merged mod collections?

**Comment by bcssov** [comment]
Patch is integrated into the merge mod when you perform basic merge so the resulting merged mod collection does not have a patch mod. Compress does though on the other hand. But a general rule is to resolve conflicts on non merged stuff and leave merged collection alone. Of course I will not stop anyone from doing so.
